src/Cpanel/CpanelAPI.php

```diff
@@ -186,7 +186,7 @@ private function uapi($module, $function, $parameters, $value)
      *
      * @return bool
      */
-    public function uapi_response_ok($cpanel_uapi_response)
+    public function uapi_response_ok($cpanel_uapi_response) // @codingStandardsIgnoreLine
```

Comment:
Yeah, eventually I'd like to refactor but it includes editing everywhere else in the code base that uses this method and possibly cloudflare/cloudflare-plugin-backend